dotcom-rendering/src/components/AdPortals.importable.tsx

```diff
@@ -86,19 +87,20 @@ export const AdPortals = () => {
 	 * client-side ad slots.
 	 */
 	useEffect(() => {
-		void getUserClient()
-			.isPremium()
-			.then((isPremium) => {
-				if (!isPremium) {
-					setAdPlaceholders(
-						Array.from(
-							document.getElementsByClassName(
-								'ad-portal-placeholder',
+		!isServer &&
```

Comment:
returning early here would reduce indentation - how do you feel about this? cf https://github.com/guardian/dotcom-rendering/pull/8702#discussion_r1311503434

Reply:
I think we do not even have to worry about it in `useEffect`, but otherwise good shout.